### PR TITLE
Various rubocop corrected items

### PR DESCRIPTION
**Pull request by johnnyshields**

### CODE REVIEW

File: lib/fat_free_crm/mail_processor/base.rb [hunk 1]
```diff
@@ -107,8 +107,8 @@ def with_new_emails
             end
           rescue Exception => e
             if %w[test development].include?(Rails.env)
```
Comment (CloCkWeRX) [inline lib/fat_free_crm/mail_processor/base.rb]: We should just use a proper logger and do 'debug' 'info' 'warning' or 'error' stuff here I think. Will raise a reminder for it.

File: spec/lib/mail_processor/sample_emails/dropbox.rb [hunk 2]
```diff
@@ -6,168 +6,168 @@
...
-LC5WanPulxOkGAIEFVJ6GATw9yCNKDaKdK1rIKJAAQkHpsRjISWDQReGHliiCG+CAGoTzAooEDVS
-DcJI6tj0CtPt2IOJBUtWMEgKEhYUnhq43KUvVEwKazTwfLmQTj8A92CKtCgTIyZ5F+6gCACyWEmD
-BU9OSkmGDlIF1GC6cAVUpiUYEkhlfCAgiLWAVEkCJXhFoUqGEwIU2QoI6qIACJIpwXoAmrQASg50
-G4LCQQBu3gjc+6lGwIA/KBPe1bOALkwasIj8wqoFFWKPqEA6QEKEmxqZQvTi1UxbEQebFRwkwJxh
-ADpMhSwQ4p2Bz9o8V84oX868ufPn0KNLn64hAgA7
-
---_006_200DA2FF7EAFC04BAD979DB9CF293BB365151E88CLEARWATERtesta_--
-END
+  plain: <<~EMAIL,
+    From: Aaron Assembler <aaron@example.com>
```
Comment (CloCkWeRX) [inline spec/lib/mail_processor/sample_emails/dropbox.rb]: This doesn't seem right, lets maybe move these to the file system or something

**Comment by johnnyshields** [comment]
@CloCkWeRX ok to merge this?

**Comment by johnnyshields** [comment]
I'm presuming this one is ok since you have follow-up tickets for your comments.

**Comment by CloCkWeRX** [comment]
Yeah, thats other stuff can be sorted later.

**Pull request merged**